### PR TITLE
Adding factories & disabled components

### PR DESCRIPTION
**Pull request by iulianpascalau**
- adding factories & disabled components

### CODE REVIEW

File: factory/statusHandlerFactory.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+		return time.Thursday, nil
+	case "friday":
+		return time.Friday, nil
+	case "saturday":
+		return time.Saturday, nil
+	case "sunday":
+		return time.Sunday, nil
+	}
+
+	return -2, fmt.Errorf("unknown day of week %s", dayOfWeek)
```
Comment (andreibancioiu) [inline factory/statusHandlerFactory.go:85]: `-2` is quite a magic value. Maybe use a named constant? Can also stay as it is, though.

File: factory/statusHandlerFactory.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+	pollingHandler, err := polling.NewPollingHandler(argsPolling)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return statusHandler, pollingHandler, pollingHandler.StartProcessingLoop()
+}
+
+func convertStringToDayOfWeek(dayOfWeek string) (time.Weekday, error) {
```
Comment (andreibancioiu) [inline factory/statusHandlerFactory.go:64]: "parseWeekday" (shorter name). Optional.

File: factory/notifiersFactory.go [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	outputNotifiers := make([]executors.OutputNotifier, 0)
+
+	logForNotifier := logger.GetOrCreate("notifiers")
+	logNotifier, err := notifiers.NewLogNotifier(logForNotifier)
+	if err != nil {
+		return nil, err
+	}
+	log.Debug("created log notifier")
+
+	// the default notifier is the one that logs in a file
```
Comment (andreibancioiu) [inline factory/notifiersFactory.go:23]: :+1:

**Review by andreibancioiu — APPROVED** [review]

File: factory/statusHandlerFactory.go (hunk 1 above)
Comment (iulianpascalau) [reply factory/statusHandlerFactory.go:85]: changed to const

**Review by andreibancioiu — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

**Pull request merged**